docker-compose.yml

```diff
@@ -9,6 +9,7 @@ services:
       - "8080:8080"
     environment: # any variable listed in catalina.sh and usually set in tomcat's setenv.sh, could be set here
       - ADMIN_PASSWORD      # value taken from env
+      - PROPS_FIREFLY_OPTIONS
```

Comment:
Just wanted to point out...
Any firefly props can be sent in via this method.  The ones declared here are either required or highly recommended when running Firefly.

Reply:
I will remove  `PROPS_FIREFLY_OPTIONS` here